src/test/TestHelper.cpp

```diff
@@ -46,6 +46,26 @@ bool adaptive::AdaptiveTree::download(const char* url,
   return nbRead == 0;
 }
 
+bool TestAdaptiveStream::download(const char* url,
```

Comment:
Why char* for URL instead of std::string? Although this is probably more of a refactoring task for another time.

Reply:
Just following the declaration here: https://github.com/xbmc/inputstream.adaptive/blob/Matrix/src/common/AdaptiveStream.h#L88
Yes it would be nice to get rid of all the C string stuff where possible at some point.